src/apps/copo_dtol_upload/utils/validation_celery_handler.py

```diff
@@ -340,11 +340,25 @@ def make_update_notifications(self, qm):
                 # this requires different logic to discriminate between symbionts
                 return False
             exsam = Sample().get_target_by_field("rack_tube", rack_tube)
-            assert len(exsam) == 1
+
+            # Check if ‘RACK_OR_PLATE_ID’ or ‘TUBE_OR_WELL_ID’ has been changed to 
+            # prevent an assertion error and an update lag if the assertion below is executed
+            if len(exsam) != 1:
+                field, value =  (
+                    ("RACK_OR_PLATE_ID", s["RACK_OR_PLATE_ID"])
```

Comment:
I think it is the rack_tube instead of RACK_OR_PLATE_ID or TUBE_OR_WELL_ID.  If new sample exists, can we just say "New Sample ( {rack_tube} ) detected " ?